rstudio/rstudio
# Data Viewer when viewing 50+ cols

### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Related to issue #13235 

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

After doing some testing and also speaking to several members of the team, we have confirmed that there is no cap on columns. So, I removed that information and replaced it with context stating that large numbers may slow things down.

I also added a new section about scrolling columns. IDK if scrolling is standard terminology for this but it does fit per its definition.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

Built docs locally using `quarto preview` - everything is displaying as expected.

---

 
<img width="649" alt="image" src="https://github.com/rstudio/rstudio/assets/31460023/5eee3973-ac7a-4022-b0fc-88b1a5a1df70">

---

![2023-06-28_12-23-02](https://github.com/rstudio/rstudio/assets/31460023/4daa323c-1334-4ff4-bbff-d71c5432cc3c)

---

### Documentation
> Specify which documentation has been added or modified and why (User Guide? Admin Guide?). If no documentation was added for a new feature, indicate why. 

See above.

### Checklist
